src/Atmos/Model/linear.jl

```diff
@@ -0,0 +1,71 @@
+abstract type AtmosLinearModel <: BalanceLaw
+end
+
+
+vars_state(lm::AtmosLinearModel, T) = vars_state(lm.atmos,T)
+vars_gradient(lm::AtmosLinearModel, T) = @vars()
+vars_diffusive(lm::AtmosLinearModel, T) = @vars()
+vars_aux(lm::AtmosLinearModel, T) = vars_aux(lm.atmos,T)
+vars_integrals(lm::AtmosLinearModel,T) = @vars()
+
+
+update_aux!(dg::DGModel, lm::AtmosLinearModel, Q::MPIStateArray, auxstate::MPIStateArray, t::Real) = nothing
+integrate_aux!(lm::AtmosLinearModel, integ::Vars, state::Vars, aux::Vars) = nothing
+flux_diffusive!(lm::AtmosLinearModel, flux::Grad, state::Vars, diffusive::Vars, aux::Vars, t::Real) = nothing
+function wavespeed(lm::AtmosLinearModel, nM, state::Vars, aux::Vars, t::Real)
+  return soundspeed_air(Float64(T_0))
```

Comment:
I would expect this to be the reference profile temperature here? @fxgiraldo or @mwarusz

Reply:
@charleskawczynski what function should I use here to get temperature of reference state?

Reply:
Don't know that it should be, just wondering.

Reply:
🤦‍♂  that's how we specify the reference state.

Reply:
Let‘s avoid `T_0` for reference state temperature (for dynamical calculations) because we use `T_0` as the thermodynamic reference temperature (a PlanetParameter) that specifies at which temperature reference specific heats are evaluated.

Reply:
The reference temperature is a function of height generally, and so would be the speed of sound. Do you want to take the `max` of this here?

Reply:
Yeah, so we'll want something like `return soundspeed_air(aux.ref_state.T)`

Reply:
We store the reference temperature, so can just use that.